config/kubernetes/kbs-config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "repository_type": "LocalFs",
+    "repository_description": {
+        "dir_path": "/opt/confidential-containers/kbs/repository"
+    },
+    "attestation_token_type": "Simple"
```

Comment:
If Attestation Service is built in, it needs another parameter https://github.com/confidential-containers/kbs/blob/main/src/api/src/config.rs#LL48C12-L48C12

Reply:
What would a sample AS config look like?

Reply:
https://github.com/confidential-containers/kbs/blob/main/src/api/src/config.rs#L73-L92